orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/CreateServerGroupStage.groovy

```diff
@@ -121,6 +121,11 @@ class CreateServerGroupStage extends AbstractDeployStrategyStage {
       additionalRollbackContext.disableOnly = true
     }
 
+    // When initiating a rollback automatically as part of deployment failure handling, only rollback to a server
+    // group that's enabled, as any disabled ones, even if newer, were likely manually marked so for being "bad"
+    // (e.g. as part of a manual rollback).
+    additionalRollbackContext.disregardDisabledCandidates = true
```

Comment:
This is to account for simpler deployment strategies like `RedBlackStrategy`, which don't plan their own rollback stage.